docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+| puget_sound | 40 | 40 | 40 | 2020-02-22T18:00 |
+
+Remember there were three series, so for 1h summaries for the last 3 hours we'd expect to have three rows per group 
+for nine total rows. But looking at the results, we only see the full three summaries for the `santa_monica` location. 
+This is because the other two series don't have data in every window of time.
+
+You can create rows for the missing summaries by calling `interpolate` after the call to aggregate. It can be used to 
+fill in default values or values based on a summary statistic of the rows like `first`, `min`, `max`, `mean`, etc. You 
+can find the [details for interpolate in the function reference]().
+
+#### Select
+The `select` function works similar to `aggregate` in that it operates on windows of time and groupings of rows.
+However, The difference is that select returns specific rows, rather than a summary data point. Here are a few 
+examples:
+
+```
+@my_data.h2o_temperature{location, state, bottom_degrees, surface_degrees, time > -3h}
+    |> select(fn: min("bottom_degrees"), by: [])
+```
+
+| location | state | bottom_degrees | surface_degrees | time
+| --- | --- | --- | --- | --- |
+| puget_sound | WA | 40 | 55 | 2020-02-22T17:10 |
+| puget_sound | WA | 40 | 54 | 2020-02-22T17:40 |
+
+Notice that all the columns are pulled through, the selected column name remains the same and note that 
+we've pulled in two records. We pull in two records because both of those have the same `min` `bottom_degrees` 
+value of `40`, which is the min for all records in that time. Notice also that `time` is the same value as those 
+individual records, which you don't see in calls to `aggregate`.
+
+You can also select in groups based on time or by other columns, just like you can with `aggregate`. Here's 
+an example using time grouping and inserting the time window for associated row selections:
+
+```
+@my_data.h2o_temperature{location, state, bottom_degrees, time > -3h}
+    |> selector(fn: max(bottom_degrees), by: ["state"], window: 1h, windowColumn: "window")
+
+// if you include window_column, the result will put the window timestamp for each selected row in that column
+```
+
+Produces this result:
+
+| location | state | bottom_degrees | time | window |
+| --- | --- | --- | --- | --- |
+| santa_monica | CA | 50 | 2020-02-22T15:01 | 2020-02-22T16:00 |
+| coyote_creek | CA | 51 | 2020-02-22T16:05 | 2020-02-22T17:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:01 | 2020-02-22T18:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:31 | 2020-02-22T18:00 |
+| santa_monica| CA | 48 | 2020-02-22T17:46 | 2020-02-22T18:00 |
+| puget_sound | WA | 40 | 2020-02-22T17:10 | 2020-02-22T18:00 |
+| puget_sound | WA | 40 | 2020-02-22T17:40 | 2020-02-22T18:00 |
+
+What this does is for each 1 hour period, it groups the rows by state and then selects the rows with the max value. 
+Because we included the `windowColumn` argument, that is included in the result with the ending timestamp of the 
```

Comment:
Is "ending timestamp" the right value to put in the window column? I'm not sure yet tbh.